main.go

```diff
@@ -22,11 +23,12 @@ import (
 )
 
 type options struct {
-	OutputLocation    string `long:"output-location" env:"OUTPUT_LOCATION" description:"local path to output"`
-	DynamoTableName   string `long:"dynamo-table-name" env:"DYNAMO_TABLE_NAME" description:"dynamo table name"`
-	S3BucketName      string `long:"s3-bucket-name" env:"S3_BUCKET_NAME" description:"s3 bucket to dump stuff into"`
-	MartaAPIKey       string `long:"marta-api-key" env:"MARTA_API_KEY" description:"marta api key" required:"true"`
-	PollTimeInSeconds int    `long:"poll-time-in-seconds" env:"POLL_TIME_IN_SECONDS" description:"time to poll marta api every second" required:"true"`
+	OutputLocation    string  `long:"output-location" env:"OUTPUT_LOCATION" description:"local path to output"`
+	DynamoTableName   string  `long:"dynamo-table-name" env:"DYNAMO_TABLE_NAME" description:"dynamo table name"`
+	S3BucketName      string  `long:"s3-bucket-name" env:"S3_BUCKET_NAME" description:"s3 bucket to dump stuff into"`
+	MartaAPIKey       *string `long:"marta-api-key" env:"MARTA_API_KEY" description:"marta api key"`
+	MartaAPIKeyFile   *string `long:"marta-api-key-file" env:"MARTA_API_KEY_FILE" description:"file containing the marta api key"`
```

Comment:
Docker secrets are loaded into files on the container, so this is a convenience option that lets us avoid having to write a start script that loads the file into an envar.